test/bench/document_bench_test.go

```diff
@@ -644,4 +676,4 @@ func benchmarkObject(cnt int, b *testing.B) {
 		})
 		assert.NoError(b, err)
 	}
-}
+}
```

Comment:
Could you keep the newline at the end of the file?

https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
Knowledge has increased. thank you!
I think it disappeared when I moved the function i made. 😅